project-resources/community-health-files-guide.md

```diff
@@ -0,0 +1,61 @@
+# Guide to Community Health Files for OpenJS Foundation Projects
+
+Community health files are documents related to project collaboration. Most commonly, these include:
+
+- `CODE_OF_CONDUCT.md`
+- `CONTRIBUTING.md`
+- `GOVERNANCE.md`
+- `SECURITY.md`
+
+> [!NOTE]
```

Comment:
Would we ever even want a non-GitHub foundation project?

Reply:
the entire document is GitHub-specific.  it's not worth it to start qualifying sections about what is GH only and what it not, unless we have actual usage/requirement for different SCM services.  if/when we have that need, we can update.

Reply:
> the entire document is GitHub-specific.

So this is interesting, and I think where we need to reach agreement first. My position is that all of those files (README, LICENSE, SECURITY, GOVERNANCE, etc.) are defacto standards, and aren't GitHub-specific. The `.github` repo on the other hand, absolutely is. I'd want us to make a distinction between the two.

> Would we ever even want a non-GitHub foundation project?

Absolutely. Why not?

Reply:
I agree with these comments (well, not Jordan's, but that seems like an unnecessary tangent to explore).  My main concern was to avoid making the guide sub-optimal by putting qualifiers everywhere or shifting content.  I expect my changes should address this issue satisfactorily.